governance/votes/Executive Vote - December 2024 out-of-schedule spell 3.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Template - [Executive Vote] Out-of-schedule Executive Vote - Stability Fee Changes, Savings Rate Changes - $date
+summary: Increase Stability Fees for multiple vault types, increase Savings Rates.
+date: $date
+address: "$spell_address"
+
+---
+# [Executive Proposal] Out-of-schedule Executive Vote - Stability Fee Changes, Savings Rate Changes - $date
+
+The Governance Facilitators, Sidestream, and Dewiz have placed an out-of-schedule executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.
+
+If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+
+- The [Stability Fees](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.3_Stability_Fee-67e40a3b-f1c2-4dc6-b502-2affeab0b232|57eaf45219bea3b430c2) for multiple vault types will be increased.
+- The [Savings Rates (DSR and SSR)](https://sky-atlas.powerhouse.io/A.4.3.1_Savings_Rates/5586fe33-b0af-41f0-9629-b1716c7c0967%7Cb3417d54) will be increased.
+
+**Voting for this executive proposal will place your MKR/SKY in support of the actions outlined above.**
+
```

Comment:
Office Hours not specified in spell sheet. Non-blocking, but should but updated on the sheet.